modules/openapi-generator/src/main/resources/C-libcurl/model-body.mustache

```diff
@@ -643,22 +642,24 @@ fail:
     keyValuePair_t *localMapKeyPair;
     cJSON_ArrayForEach({{{name}}}_local_map, {{{name}}})
     {
+		cJSON *localMapObject = {{{name}}}_local_map;
```

Comment:
Hi

Why "{{{name}}}_local_map" is changed to "cJSON *localMapObject" ?

Reply:
Later on, the code did:
`cJSON_AddItemToObject({{{name}}},"", localMapObject);`
meaning that there was an in item containing an item etc. The JSON for 
`{"fruit" : "apple"}`
 would become 
`{"" : {"fruit" : "apple"}}`
 (that's why I verified my JSON against the one produced by other generators - see the beginning of my PR message)

I needed to use `{{{name}}}_local_map` in the code, for example a few lines later where I do:
`localMapKeyPair = keyValuePair_create(strdup(localMapObject->string),strdup(localMapObject->valuestring));`
but since I was into an `{{#items}}`, the content of `{{{name}}}` was different.
Thus, I provided an alias for `{{{name}}}_local_map`.

Reply:
OK. I got it.